mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/RabbitMQEventBusTest.java

```diff
@@ -482,8 +535,11 @@ void dispatchShouldWorkAfterNetworkIssuesForNewKeyRegistration() {
 
                 rabbitMQExtension.getRabbitMQ().pause();
 
-                assertThatThrownBy(() -> eventBus.dispatch(EVENT, NO_KEYS).block())
-                    .isInstanceOf(RabbitFluxException.class);
+                try {
```

Comment:
Can you add one test where you assert that `dispatch(...)` throws when RabbitMQ is stopped?

Reply:
With the current implementation, even when RabbitMQ stopped, there is no exception thrown because the dispatch operation is succeed right after an event is putted inside the SynchronousQueue. 

It doesnt care whether RabbitMQ Sender success to send events or not

Reply:
Isn't this behaviour `bad`?

Reply:
I removed try catch, check it out 0de8e9d